cmd/agent/agent.go

```diff
@@ -37,9 +44,27 @@ func NewAgentCmd() *cobra.Command {
 
 	agentCmd.AddCommand(startCmd)
 
+	agentCmd.AddCommand(runOnceCmd)
+
 	return agentCmd
 }
 
+func runOnce(cmd *cobra.Command, args []string) {
+	var err error
+
+	cfg, err := agent.DefaultConfig()
+	if err != nil {
+		log.Fatal("Failed to create the agent configuration: ", err)
+	}
+
+	cfg.DefinitionsPath = args[0]
+
+	_, err := agent.NewWithConfig(cfg)
+	if err != nil {
+		log.Fatal("Failed to create the agent: ", err)
+	}
```

Comment:
Once you address my previous comment, all of these lines will no longer be needed: we don't need to create an Agent instance, we need to create a Checker one.